nesta_ds_utils/nesta_io.py

```diff
@@ -0,0 +1,41 @@
+
+from io import BytesIO
+from typing import Union
+import pathlib
+import zipfile
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Stylistic: The style guide doesn't specify this explicitly, but we normally treat imports and globals as top level definitions and have 2 lines between them.